### ISSUE TITLE
GDPR compliance

### ISSUE THREAD

**Issue by kurumbus** [websquared/lfc#104]
You are a better judge on  at what point should we need this, but the main thing I think is the ability for a user to delete their account (for reals, with deleting or anonymizing personal data, not just soft delete)

It has all kinds of implications: if an instructor deletes account at what point shall we delete a course? (especially if somebody paid for it) and other business questions. Just deleting a record is, of course, a bad option because all sorts of foreign key constraints shall fall off.

**Comment by websquared** [comment]
Yes, we'll have to think about this but not for now. We not targeting Europe yet, just US and Canada. 
But it's a valid point.
